extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonUtils.java

```diff
@@ -93,6 +93,12 @@ private static long getConnectionDelay(OidcCommonConfig oidcConfig) {
                 : 0;
     }
 
+    public static long getConnectionDelayInMillis(OidcCommonConfig oidcConfig) {
```

Comment:
@Yelzhasdev Or if you prefer to keep this function for millis specifically then still please consider reusing the one above it and only div by 1000 here...

Reply:
Sorry, my bad, now I just reuse getConnectionDelay and multiply it by 1000 to get a millis

Reply:
Good idea about multiply, glad you did not decide to go with mine about div :-)